.github/workflows/ci.yml

```diff
@@ -157,6 +235,47 @@ jobs:
             orm:
               name: 'active_record'
               version: '5.2'
+          - ruby: '3.1'
+            orm:
+              name: 'active_record'
+              version: '4.2'
+          - ruby: '3.1'
+            orm:
+              name: 'active_record'
+              version: '5.0'
+          - ruby: '3.1'
+            orm:
+              name: 'active_record'
+              version: '5.1'
+          - ruby: '3.1'
+            orm:
+              name: 'active_record'
+              version: '5.2'
+          - ruby: '3.1'
+            orm:
+              name: 'active_record'
+              version: '6.0'
```

Comment:
According to [the Ruby & Rails Compatibility Table](https://www.fastruby.io/blog/ruby/rails/versions/compatibility-table.html), Rails 6.0 does not officially support Ruby 3.0 or above.